content/applications/inventory_and_mrp/inventory/product_management/product_tracking/packaging.rst

```diff
@@ -140,4 +140,50 @@ used for the product. The chosen packaging is displayed on the :abbr:`SO (Sales
       :align: center
       :alt: Assign packagings on the Sales Order Line.
 
+.. _inventory/product_management/packaging-route:
+
+Routes for packaging
+====================
+
+When receiving packagings, by default, they follow the warehouse's :doc:`configured reception route
+<../../shipping_receiving/daily_operations/shipments_deliveries>`. To **optionally** set up a
+packaging-specific route, go to :menuselection:`Inventory app --> Configuration --> Routes`.
+
+.. important::
+   The *Product Packagings*, *Storage Locations*, and *Multi-Step Routes* features (found by going
+   to :menuselection:`Inventory app --> Configuration --> Settings`) **must** be activated, and
+   saved.
+
+.. seealso::
+   :doc:`../../warehouses_storage/inventory_management/use_routes`
+
+Create route
+------------
+
+On the :guilabel:`Routes` page, click :guilabel:`New`, or select a route that is **not** for a
+warehouse. Next, in the :guilabel:`Applicable on` section, tick the :guilabel:`Packagings` checkbox.
+
+.. figure:: packaging/route.png
+   :align: center
+   :alt: Create route for a packaging.
+
+   Route with "Packagings" selected, with "Products" and "Warehouses" not selected.
+
+Apply route on packaging
+------------------------
+
+Then, to apply the route, go to :menuselection:`Inventory app --> Products --> Products`, and
+select the product that uses packaging.
+
+In the product form, switch to the :guilabel:`Inventory` tab. In the :guilabel:`Packaging` section
+that contains :ref:`configured packagings <inventory/product_management/packaging-setup>`, click the
+:icon:`oi-settings-adjust` :guilabel:`(settings)` icon. Tick the :guilabel:`Routes` checkbox to make
```

Comment:
We call this same icon a few different things in this doc.

I believe it falls under the "common use" rule that "⚙️ Action menu" and "☁️ Save" icons do (instead of calling them (gear) and (cloud), respectively), and would be called "additional options" icon or menu. 

cc: @samueljlieber please validate on TR, thanks!

Reply:
Hmm, it might be adjust?
![image](https://github.com/user-attachments/assets/018cea72-2946-448e-a1a9-179df971d1b8)

I'll make sure to fix all the icons in other instances of the doc!

Reply:
> Hmm, it might be adjust?

Again I'm pretty sure it's "Additional options" as that's what it's commonly referred to by instructors and the community.

Reply:
Yes I am recommending "(additional options)" as the label for this icon
```suggestion
:icon:`oi-settings-adjust` :guilabel:`(additional options)` icon. Tick the :guilabel:`Routes` checkbox to make
```